src/scenes/ContactUsRouter/ContactUsHome.tsx

```diff
@@ -56,24 +64,45 @@ export default function ContactUsView(): JSX.Element {
         <i>`&quot;`{appVersion || 'n/a'}`&quot;`</i>
       ) as string,
       buttonText: strings.REPORT_PROBLEM,
-      link: `${docLinks.report_a_problem}?build=${appVersion || ''}`,
+      onClick: () => window.open(`${docLinks.report_a_problem}?build=${appVersion || ''}`),
     },
     {
       icon: 'sparkles',
       title: strings.TITLE_REQUEST_FEATURE,
       description: strings.DESCRIPTION_REQUEST_FEATURE,
       buttonText: strings.REQUEST_FEATURE,
-      link: docLinks.request_a_feature,
+      onClick: () => window.open(docLinks.request_a_feature),
     },
     {
       icon: 'mail',
       title: strings.CONTACT_US,
       description: strings.formatString(strings.CONTACT_US_DESCRIPTION, getHelpEmail()) as string,
       buttonText: strings.CONTACT_US,
-      link: docLinks.contact_us,
+      onClick: () => window.open(docLinks.contact_us),
     },
   ];
 
+  const jiraListItems = useMemo(
+    () =>
+      (types ?? []).map(
+        (item): ListItemContent => ({
+          title: item.name,
+          description: item.description,
+          buttonText: item.name,
+          onClick: () => goToContactUsForm(item.requestTypeId),
+        })
+      ),
+    [types]
+  );
+
+  const listItemContent = useMemo(() => {
+    if (featureEnabled && jiraListItems.length > 0) {
+      return [knowledgeBaseItem, ...jiraListItems];
+    } else {
+      return [knowledgeBaseItem, ...defaultListItems];
+    }
+  }, [featureEnabled, jiraListItems]);
```

Comment:
Since `knowledgeBaseItem` is using localized strings, you also need to include `activeLocale` in the deps here